drivers/vl6180x/Kconfig

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) 2021 Gunar Schorcht
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_VL6180X
+    bool "VL6180X Ranging and Ambient Light Sensing (ALS) module"
+    depends on TEST_KCONFIG
+    depends on HAS_PERIPH_I2C
+    select MODULE_PERIPH_I2C
+    select MODULE_ZTIMER
+    select MODULE_ZTIMER_MSEC
+    help
+        Driver for the ST VL6180X Ranging and Ambient Light Sensing (ALS) module
+
+if MODULE_VL6180X
+
+config MODULE_VL6180X_RNG
```

Comment:
I'm a bit puzzled by the RNG abbreviation. I find it a bit disturbing at first glance.
I am not asking for a change here but I am wondering what other think about it.

Reply:
Ah ok, could be misinterpreted as random number generator. I din't think about it, I was just looking for short wording.